packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/stack.md

```diff
@@ -2,4 +2,31 @@
 id: Stack
 section: layouts
 ---
-Use a **Stack** layout to position items vertically in a container, with one item filling the available vertical space.
+The **stack** layout positions items vertically, with one or more items filling the available vertical space.
+
+## Elements
+
+<img src="./img/stack-example.png"  alt="PatternFly website screenshot showing modal created with bullseye layout."  width="3330px"/>
+
+1. **Stack**: The stack container, which holds all stack items.
+1. **Stack item**: An individual section of content to be placed in the stack. 
+
+## Usage
+
+### When to use stack vs other layouts 
+
+The stack layout does not wrap. If you need a layout that wraps, instead choose the flex, gallery, level, or split layout instead.
```

Comment:
I feel like the way this is structured doesn't align well with the header. I'd say something like "User the stack layout when you do not need your layout to wrap. If you need a layout that wraps, instead.. etc"